arkiverer/src/main/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/service/TaskListService.kt

```diff
@@ -9,17 +9,19 @@ import no.nav.soknad.arkivering.soknadsarkiverer.kafka.KafkaPublisher
 import no.nav.soknad.arkivering.soknadsarkiverer.service.fileservice.FilesAlreadyDeletedException
 import no.nav.soknad.arkivering.soknadsarkiverer.supervision.ArchivingMetrics
 import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.beans.factory.config.ConfigurableBeanFactory.SCOPE_SINGLETON
 import org.springframework.context.annotation.Scope
 import org.springframework.stereotype.Service
 import java.time.Instant
 import java.time.LocalDateTime
 import java.util.concurrent.Semaphore
 
-@Scope(SCOPE_SINGLETON) // This is a stateful component so it must be singleton
-@Service
-class TaskListService(
+
+open class TaskListService(
 	private val archiverService: ArchiverService,
+	private val startUpSeconds: String,
```

Comment:
Can we change the type to Long instead of doing awkward type casting whenever we use it?